ophyd/tests/test_signal.py

```diff
@@ -695,3 +695,71 @@ def test_import_ro_signal_class():
     from ophyd.signal import SignalRO as SignalRoFromModule
 
     assert SignalRoFromPkg is SignalRoFromModule
+
+
+def test_numeric_value_info(fake_motor_ioc, cleanup):
+    pvs = fake_motor_ioc.pvs
+    sig = EpicsSignal(write_pv=pvs["setpoint"], read_pv=pvs["readback"], name="motor")
+    sig_desc = sig.describe()["motor"]
+    assert sig_desc["dtype"] == "number"
+    assert sig_desc["shape"] == ()
+
+    sig = EpicsSignal(
+        write_pv=pvs["setpoint"],
+        read_pv=pvs["readback"],
+        name="motor",
+        value=NumericValueInfo(float),
+    )
+    sig_desc = sig.describe()["motor"]
+    assert sig_desc["dtype"] == "float64"
+    assert sig_desc["shape"] == ()
+
+    original = Signal(name="original")
+    original_desc = original.describe()["original"]
+    assert original_desc["dtype"] == "float64"
```

Comment:
This is out of the schema for things that can be in descriptors: https://github.com/bluesky/event-model/blob/a1fdaec0c45b89e513d0be9dba0bfc453982e5bb/src/event_model/schemas/event_descriptor.json#L51-L62

Reply:
Ok so newest version has `dtype` and `dtype_numpy`